usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -112,11 +112,21 @@ def filters(self):
                 "name": "award_type_codes",
                 "type": "array",
                 "array_type": "enum",
-                "enum_values": list(loan_type_mapping.keys()),
+                "enum_values": sorted(list(loan_type_mapping.keys())),
```

Comment:
I see this a lot.  There's no need to change it, but `sorted` returns a list.  Also, the default iterator behavior of a dictionary returns only the keys so this could be simplified down to `sorted(loan_type_mapping)`.

Reply:
Will make the change since I will be making other changes anyways